GeneratorInterface/LHEInterface/test/testMerge.log

```diff
@@ -0,0 +1,153 @@
+Number of particles = 12
```

Comment:
hi, is this log file needed?

Reply:
It is used by the unit test. The test compares the output of the test job to this log file and if there is a difference, the test fails.